packages/rest/test/acceptance/bootstrapping/application.acceptance.ts

```diff
@@ -5,35 +5,40 @@
 
 import {expect, supertest} from '@loopback/testlab';
 import {Constructor, Provider, inject} from '@loopback/context';
-import {Application, DefaultSequence, CoreBindings} from '../../..';
+import { DefaultSequence, RestBindings, RestServer } from '../../..';
+import {Application} from '@loopback/core';
 
 describe('Bootstrapping the application', () => {
   context('with a user-defined sequence', () => {
     let app: Application;
+    let server: RestServer;
     before(givenAppWithUserDefinedSequence);
 
     it('binds the `sequence` key to the user-defined sequence', async () => {
-      const binding = await app.get(CoreBindings.SEQUENCE);
+      const binding = await server.get(RestBindings.SEQUENCE);
       expect(binding.constructor.name).to.equal('UserDefinedSequence');
     });
 
     function givenAppWithUserDefinedSequence() {
       class UserDefinedSequence extends DefaultSequence {}
-      app = new Application({
+      app = new Application();
+      server = new RestServer(app, {
         sequence: UserDefinedSequence,
```

Comment:
Server-specific sequence makes sense to me. I am not sure if you made this change in all other places too, e.g. `app.sequence` should become `server.sequence` too, etc. Could you please check?

Reply:
I think this is the only location where we directly add sequence in this way.
I did a search across `./packages` for `app.sequence` without result.

I removed the sequence handler from `application.ts` and moved it to `rest-server.ts` as a part of the refactor, so any calls to `app.sequence` shouldn't compile anyway.

Reply:
Hmm, this is bringing sequence back to `ApplicationConfig`. Since the sequence is transport-specific, and there may be multiple sequences (a REST sequence, a gRPC sequence), I think it's a bad idea to configure the sequence this way.

I think this is probably caused by misconfiguration of the config argument of RestServer constructor as I commented earlier.